### ISSUE TITLE
dataset 、network

### ISSUE THREAD

**Issue by wudami** [qizhuzhuang/PTDCC#1]
Hello, I have a few questions I'd like to ask.

In your work, you mentioned using 3,019 images as the training set and 20 images as the test set. Can your dataset be made publicly available?

In your code, I didn't see the part related to generating the learn_mask. Is this part implemented using the second stage of UDIS++?

**Comment by qizhuzhuang** [comment]
1. The filtered dataset we used has been published at the following link: https://pan.quark.cn/s/f0c4195d5a76, code: LQSt
2. Yes, the stitching masks were generated using the second stage of UDIS++ without any additional modifications on our part.

**Comment by wudami** [comment]
Thank you for your answer. Could you please let me know if the evaluation metrics you mentioned in the article are reflected in the code? I might have missed them.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "qizhuzhuang/PTDCC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年4月28日(星期一) 下午5:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [qizhuzhuang/PTDCC] dataset 、network (Issue #1)



 qizhuzhuang left a comment (qizhuzhuang/PTDCC#1)
  
The filtered dataset we used has been published at the following link: https://pan.quark.cn/s/f0c4195d5a76, code: LQSt
 
Yes, the stitching masks were generated using the second stage of UDIS++ without any additional modifications on our part.

**Comment by qizhuzhuang** [comment]
Sorry for the oversight. You can now find the corresponding evaluation metric calculations in cdcs.py.
